app.js

```diff
@@ -3,21 +3,29 @@ const express = require("express");
 const {
     getArticles,
     getArticleById,
+    getArticleComments,
 } = require("./controllers/articlesController");
 const { getTopics } = require("./controllers/topicsController");
 
 const {
     sqlErrorHandler,
     customErrorHandler,
+    pathNotFoundHandler,
 } = require("./controllers/errorHandlerControllers");
-const errorHandlers = [sqlErrorHandler, customErrorHandler];
+const errorHandlers = [
+    pathNotFoundHandler,
+    sqlErrorHandler,
+    customErrorHandler,
+];
 
 const app = express();
 
 app.get("/api/articles", getArticles);
 
 app.get("/api/articles/:article_id", getArticleById);
 
+app.get("/api/articles/:article_id/comments", getArticleComments);
+
 app.get("/api/topics", getTopics);
 
 app.use(...errorHandlers);
```

Comment:
I had to check the docs to see if this would work ... and it does! 
It feels a little less declarative than adding each one but I dont mind the syntax at all, perhaps it could be clearer if you were expecting your error handlers as an array, but then that leads to inconsistent exports, or maybe even something as simple as declaring the array just before you spread them in might make it clearer what is going on here :)